db/statistics.go

```diff
@@ -201,8 +203,9 @@ func WriteValidatorTotalPerformance(day uint64) error {
 
 	if err != nil {
 		return fmt.Errorf("error retrieving required data: %v", err)
-	} else if !exported.CurrentCLRewards || !exported.CurrentElRewards || !exported.LastClRewards || !exported.LastElRewards {
-		return fmt.Errorf("missing required export: cur cl rewards: %v, cur el rewards: %v, last cl rewards: %v, last el rewards: %v", !exported.CurrentCLRewards, !exported.CurrentElRewards, !exported.LastClRewards, !exported.LastElRewards)
+	} else if !exported.LastTotalPerformance || !exported.CurrentCLRewards || !exported.CurrentElRewards || !exported.CurrentSyncDuties || !exported.CurrentFailedAttestations {
```

Comment:
As discussed, this check will fail for day 0 as you now have the following logic:
Export Total Performance for day 0 only if Total Performance for day 0 is exported. Mind the `cur.day - 1` here: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/7cf3a5e2ebb03ab20ba5dc1d01981f2765d62533/db/statistics.go#L200

Reply:
fixed